dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -1000,3 +1000,20 @@ CWWKD1100.cursor.requires.sort.explanation=Cursor-based pagination requires \
 CWWKD1100.cursor.requires.sort.useraction=Remove the ORDER BY clause from the \
  Query annotation value if one is found there. Add an OrderBy annotation or a \
  repository method parameter to specify the sort criteria.
+
+CWWKD1101.attr.subset.mismatch=CWWKD1101E: The {0} return type of the \
```

Comment:
> The {0} return type of the {1} method of the {2} repository attempts to return results as the subset of entity attributes that are represented by the {3} Java record type, expecting the {4} entity to have attributes named: {5}.

can this sentence be either simplified or broken up? For example, would something like the following be oversimplified?

> The {0} return type of the {1} method of the {2} repository returns the subset of entity attributes that are represented by the {3} Java record type. This method expects the {4} entity to the following attributes: {5}.

In this context, does "expects" mean "requires"?

Reply:
> In this context, does "expects" mean "requires"?

It would be more accurate to say that in the opposite direction, that the Java record type is required to represent a subset of entity attributes.  The entity is what determines the entity attributes, and the record needs to choose a subset of those.

Reply:
I made an attempt to improve the message, partially using what you wrote. Please see https://github.com/OpenLiberty/open-liberty/pull/30472/commits/fee8213a303723b4d1edc5585ccf76423d929ee2